lib/hydra/derivatives/services/capability_service.rb

```diff
@@ -0,0 +1,17 @@
+require 'open3'
+
+module Hydra::Derivatives
+  class CapabilityService
+    attr_accessor :ffmpeg_output
+    def capture_output
+      @ffmpeg_output = Open3.capture3('ffmpeg -codecs').to_s
```

Comment:
This is a personal preference, and not a blocking problem; I prefer to set instance variables as part of the initialize (and disallow instance variables to be set once the object is instantiated). This reduces mutability and better adheres to data encapsulation.